wasatch/DeviceFinderUSB.py

```diff
@@ -36,7 +37,9 @@ class DeviceFinderUSB(object):
     def __init__(self):
         self.system = platform.system()
```

Comment:
`platform.system()` has no cost, why not call it every time ?

ex:
```
elif platform.system() == "Darwin"
```

That may make the code more portable in the future.